microsoft-edge/webview2/concepts/overview-features-apis.md

```diff
@@ -1347,6 +1347,32 @@ The history methods allow back and forward navigation in WebView2, and the histo
 ---
 
 
+<!-- ------------------------------ -->
+#### Navigation kind
```

Comment:
Is the new Navigation kind section in right spot in outline?
Rendered section:
https://review.learn.microsoft.com/en-us/microsoft-edge/webview2/concepts/overview-features-apis?branch=pr-en-us-2733&tabs=dotnetcsharp#navigation-kind

Reply:
This being in the "Navigate to pages and manage loaded content" LGTM.